### PR TITLE
nvimpager: 0.9 -> 0.10

### PR DESCRIPTION
**Pull request by lucc**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/nvimpager/default.nix [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-  nativeBuildInputs = [ pandoc ];
+  nativeBuildInputs = [ scdoc ];
 
   makeFlags = [ "PREFIX=$(out)" ];
-  buildFlags = [ "nvimpager.configured" ];
+  buildFlags = [ "nvimpager.configured" "nvimpager.1" ];
   preBuild = ''
     patchShebangs nvimpager
     substituteInPlace nvimpager --replace ':-nvim' ':-${neovim}/bin/nvim'
+    substituteInPlace makefile --replace '$(shell git log -1 --no-show-signature --pretty="%ct")' 1623019602
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/nvimpager/default.nix:29]: Do we need to update this number?

Comment (lucc) [reply pkgs/tools/misc/nvimpager/default.nix:29]: That number is the output of the shell command that it replaces. The makefile assumes that it is run in a git checkout and inspects the last commit date. That is the unix timestamp of the commit.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126027` at 5175b549 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>nvimpager</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`checkPhase` should probably contain `runHook preCheck` and `runHook postCheck`.

Near pkgs/tools/misc/nvimpager/default.nix:34:3:
```
   |
34 |   checkPhase = ''script -c "busted --lpath './?.lua' test"'';
   |   ^
```
</li>
  </ul>
</details>

**Comment by lucc** [comment]
I added extra commits for the comments above. We can squash them after a positive review.

**Pull request merged**